src/ameba/rule/lint/spec_focus.cr

```diff
@@ -65,7 +65,7 @@ module Ameba::Rule::Lint
       return unless node.block
 
       arg = node.named_args.try &.find(&.name.== "focus")
-      return unless arg
+      return if arg.nil? || arg.value.is_a?(Crystal::Call)
```

Comment:
Will this resolve the OP's example, when the value is an argument?

```crystal
def foo(focus = false)
  it "foo", focus: focus do
  end
end
```

Reply:
It doesn't matter, this rule operate on calls only.

Reply:
But it means it does not resolve #605, right? Sorry, I may have missed something.

Reply:
It does, you have the test added to cover it.

Reply:
I've fixed the code and modified the test case to cover this exact code path.